Sources/ClientRuntime/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -80,7 +80,7 @@ public extension DefaultSDKRuntimeConfiguration {
     static func makeClient(
         httpClientConfiguration: HttpClientConfiguration = defaultHttpClientConfiguration
     ) -> HTTPClient {
-        #if os(iOS) || os(tvOS) || os(watchOS) || os(visionOS) || os(macOS)
+        #if os(iOS) || os(tvOS) || os(watchOS) || os(visionOS)
```

Comment:
Let's split this out to a separate PR since it is unrelated to the fix below and these are both significant changes.

Reply:
Ok 👍